rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -293,8 +280,7 @@ private static Stream<Arguments> provideSaveFederationTestArguments() {
     @Test
     void saveNewFederation_before_RSKIP123_should_not_save_null() {
         Repository repository = mock(Repository.class);
-        activations = mock(ActivationConfig.ForBlock.class);
-        when(activations.isActive(ConsensusRule.RSKIP123)).thenReturn(false);
+        activations = ActivationConfigsForTest.orchid().forBlock(0L);
```

Comment:
```suggestion
        ActivationConfig.ForBlock orchidActivations = ActivationConfigsForTest.orchid().forBlock(0L);
```
wdyt? To be consistent with other tests